Makefile

```diff
@@ -201,6 +219,9 @@ lint-full: $(GOLANGCI_LINT) ## Run slower linters to detect possible issues
 	cd $(E2E_FRAMEWORK_DIR); $(GOLANGCI_LINT) run -v --fast=false
 	cd $(CAPD_DIR); $(GOLANGCI_LINT) run -v --fast=false
 
+apidiff: $(GO_APIDIFF) ## Check for API differences
+	$(GO_APIDIFF) $(shell git rev-parse origin/master) --print-compatible
```

Comment:
`go-apidiff` wasn't added here originally because it doesn't work on dirty trees and it could be confusing when used in development

Reply:
Yeah, I found out the hard way. Would still like a way to run it locally.
Was *surprising* to see it.

Reply:
Maybe let's remove the doc string so it doesn't show up in help?